### PR TITLE
fix: enhancements to install dependencies for mono repo

### PR DESCRIPTION
**Pull request by manisfdcsfdx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -459,12 +464,14 @@ OPTIONS
                                                     compilation of the apex in the entire org is triggered
   --apiversion=apiversion                          override the api version used for api requests made by this command
 
+  --createdwithdependency=createdwithdependency    when installing with .LATEST buildnumber, pick the lastest package created with dependencies.
```
[the remark this answers is not in the document]
Comment (manisfdcsfdx) [reply README.md:467]: This use case is final build validation where i want all dependencies that are created with depencency check. Less see value in few package only checking this.

[the remark this answers is not in the document]
Comment (azlam-abdulsalam) [reply README.md:467]: if we are not allowing pattern here, this is a boolean flag, lets rename  to --usedependencyvalidatedpackages with description  "Use only dependency validated packages that match the version number"

[the remark this answers is not in the document]
Comment (manisfdcsfdx) [reply README.md:467]: Makes sense will update

[the remark this answers is not in the document]
Comment (manisfdcsfdx) [reply README.md]: flag name updated

**Comment by azlam-abdulsalam** [comment]
@manisfdcsfdx Can you update it with conflicts resolved?

**Pull request merged**